scss/_patterns_contextual-menu.scss

```diff
@@ -83,6 +83,8 @@
   }
 
   .p-contextual-menu__toggle {
+    // Remove margin bottom for the dropdown menu to be aligned with the toggle
+    margin-bottom: 0;
```

Comment:
I'm not sure if we can do that. In our example `p-contextual-menu__toggle` is used on the button, and in such case it would override the default margin on the button.

The fact that button has contextual menu should not affect its margin.

I guess this was the main reason to use JS for positioning - so we can calculate top position regardless of margin that element having contextual menu may have.